example/lambdas/s3AccessTest/index.js

```diff
@@ -74,8 +74,21 @@ async function testList(s3, params) {
  * @returns {undefined} - does not return a value
  */
 async function handler(event) {
-  const { credentials, testName, ...params } = event;
-  const s3 = new S3({ credentials: new Credentials(credentials) });
+  const {
+    credentials: {
+      accessKeyId,
+      secretAccessKey,
+      sessionToken
+    },
+    testName,
+    ...params
+  } = event;
+
+  const s3 = new S3({
```

Comment:
Perhaps this is intentional but it seems like the object we're passing to the S3 constructor has changed? Before there was a `credentials` key which had a `Credentials` object but now we're passing the access keys / token in as the object keys?

Reply:
It is intentional, reading the aws documentation it says you shouldn't have to create Credentials objects yourself. https://docs.aws.amazon.com/AWSJavaScriptSDK/latest/AWS/Credentials.html
```
Note that this class typically does not need to be constructed manually, as the AWS.Config and AWS.Service classes both accept simple options hashes with the three keys. These structures will be converted into Credentials objects automatically.
```
I had already switched it out of the spec code, but forgot to take it out of the test lambda 

It's also the piece of code that would give the hmac strange error if it was created with bad values for the 3 required keys.  So I just thought this was safer.
```
    at new Hmac (crypto.js:117:16)
    at Object.createHmac (crypto.js:643:10)
    at Object.hmac (/Users/savoie/projects/cumulus/cumulus/example/node_modules/aws-sdk/lib/util.js:400:30)
```